lib/chef/knife/vsphere_vm_clone.rb

```diff
@@ -597,7 +597,13 @@ def generate_clone_spec(src_config)
 
     if get_config(:customization_vlan)
       vlan_list = get_config(:customization_vlan).split(',')
-      networks = vlan_list.map { |vlan| find_network(vlan) }
+      networks =
+        if get_config(:customization_sw_uuid)
+          sw_uuid = get_config(:customization_sw_uuid)
```

Comment:
Total random thought that I'll throw in here before I get back to my family, but rather than an option, what about letting people pass the name with the dswitch annotated instead?

--cvlan NETWORK1@dswitch1,NETWORK2@AA-BB-CC,NETWORK3

It could be done within `find_network`, though `find_available_hosts` would need to know about it.

Just a crazy thought that I haven't thought through.